ensureconda/cli.py

```diff
@@ -62,172 +49,5 @@ def ensureconda_cli(mamba, micromamba, conda, conda_exe, no_install):
         sys.exit(1)
 
 
-def ext_candidates(fpath: str) -> Iterator[str]:
```

Comment:
The moved out to either installer or resolve to keep the responsibilities of the cli module clear